common/SmokeTests/SmokeTest/Program.cs

```diff
@@ -247,7 +253,7 @@ IEnumerable<Assembly> LoadAssemblies(Assembly rootAssembly, string assemblyFileM
     {
         var assembly = assembliesToProcess.Pop();
 
-        if ((!assembly.FullName.StartsWith("System.")) && (!processedAssemblies.Contains(assembly.FullName)))
+        if ((ShouldLoadAssembly(assembly)) && (!processedAssemblies.Contains(assembly.FullName)))
```

Comment:
For readability, I'm always happen to see extraneous parens...but around single operands seems excessive and actually hurts readability, IMO:
```suggestion
        if (ShouldLoadAssembly(assembly) && !processedAssemblies.Contains(assembly.FullName))
```

Reply:
I'm a fan of explicit parenthesis, personally.   Since the rest of the code is written in this style, I'm going to keep the original formatting for consistency with surrounding code.   If you'd prefer to reformat, I'm not strongly opposed but ask that you do a pass over the entire file in a separate PR.

Reply:
Not passionate enough to reformat no. But as pointed out, I'm also a fan of explicit parens, but around a single operand? It adds nothing. It just makes reviewers have to parse even more parens that add no value. Around something like `((foo && bar) || baz)` sure - that's the behavior, but the parens make it more obvious for readers, especially if they don't know operator precedence as well. But taking that same example, how does parens around `baz` provide value and instead just not make it less readable: `((foo && bar) || (baz))`?